lib/handshake_crypto.mli

```diff
@@ -3,3 +3,6 @@ open State
 val derive_master_secret : Core.tls_before_13 -> session_data -> Cstruct.t -> Cstruct.t list -> Core.master_secret
 val initialise_crypto_ctx : Core.tls_before_13 -> session_data -> (crypto_context * crypto_context)
 val finished : Core.tls_before_13 -> Ciphersuite.ciphersuite -> Cstruct.t -> string -> Cstruct.t list -> Cstruct.t
+
+val pseudo_random_function : Core.tls_before_13 -> Ciphersuite.ciphersuite ->
+  int -> Cstruct.t -> string -> Cstruct.t -> Cstruct.t
```

Comment:
There are three `Cstruct.t`/`string` in a row so maybe a comment or labels would be nice.
```suggestion
(** [pseudo_random_function version cipher length secret label seed] *)
val pseudo_random_function : Core.tls_before_13 -> Ciphersuite.ciphersuite ->
  int -> Cstruct.t -> string -> Cstruct.t -> Cstruct.t
```